### DISCUSSION TITLE
[request]Hand drawing approximate graphics, automatically converted to standard graphics

### DISCUSSION THREAD

**Discussion by bdbox1** [pubpub-zz/ppInk#296] [Ideas]
1、Draw approximate shapes with your free hand, for example, draw an approximate circle;
2、Left mouse button hold pause for 2s;
3、Automatically converts the drawn figure to a standard circle.
![1703247778420](https://github.com/pubpub-zz/ppInk/assets/8018224/54fbdcff-3069-465d-a1dc-e13884afabb1)

**Comment by pubpub-zz** [comment]
This could be a nice feature quite tough to code. Also it may be difficult to size/position properly the object. however an interesting idea

**Comment by pubpub-zz** [comment]
I personally will not implement this. No PR to propose it

**Comment by pubpub-zz** [comment]
since no update I convert this as an idea
